aptos-move/framework/aptos-framework/sources/aggregator/aggregator.spec.move

```diff
@@ -6,20 +6,24 @@ spec aptos_framework::aggregator {
     spec add {
         // TODO: temporary mockup.
         pragma opaque;
+        aborts_if false;
     }
 
     spec sub {
         // TODO: temporary mockup.
         pragma opaque;
+        aborts_if false;
```

Comment:
Same as the above. This native function can actually abort. Please refer to the comment in aggregator.move.

Reply:
What is updated?

According to the comment in the code, this function "Aborts on going below zero.".